code/src/containers/dashboard-orders-filter-drawer/hooks/use-filtered-admin-orders/useFilteredAdminOrders.test.ts

```diff
@@ -194,4 +182,31 @@ describe("useFilteredAdminOrders", () => {
       expect(result.current.orders).toEqual([]);
     });
   });
+
+  test("does not set default sort parameter if already present", () => {
+    const searchParams = new URLSearchParams("sort=total,asc");
+    (useSearchParams as jest.Mock).mockReturnValue([
+      searchParams,
+      mockSetSearchParams
+    ]);
+
+    renderHook(() => useFilteredAdminOrders());
+
+    expect(mockSetSearchParams).not.toHaveBeenCalled();
+    expect(searchParams.get("sort")).toEqual("total,asc");
+  });
+
+  test("sets page to 1 when page parameter is absent", () => {
+    const { result } = renderHook(() => useFilteredAdminOrders());
+    expect(result.current.page).toBe(1);
+  });
+
+  test("sets page to the value of the page parameter when present and valid", () => {
+    (useSearchParams as jest.Mock).mockReturnValue([
+      new URLSearchParams("page=2"),
+      mockSetSearchParams
+    ]);
```

Comment:
Please create reusable util for following snippet:

```tsx
(useSearchParams as jest.Mock).mockReturnValue([
      searchParams,
      mockSetSearchParams
    ]);

    renderHook(() => useFilteredAdminOrders());
```

Reply:
Reusable function created but without renderHook, because I use it also in beforeEach block, where this render would be redundant
Also renderHook is present in another tests